README.md

```diff
@@ -70,15 +70,15 @@ Mode 2 can be used to verify the Suyama A residue stored in a verifiable delay f
 ```bash
 cofact -cpr F14.proof 14
```

Comment:
One other thing I think would be worth doing before you finish this set of modifications to the main branch, is to also add the F14.proof to the `main` branch, to allow these suggested tests of the -cpr and -upr modes to work ‘right out of the box’, without the end user having to go off and find proofs for F17 and upward.

I had originally added the F17 proof to the `cxc` branch (which was 100K in size, and the smallest that the unmodified mprime would create), but the F14 proof at 12K is a nice size for a test file – the calculation of the F14 residues are non-trivial and there’s a single known factor so the syntax of
```bash
cofact [-cpr or -upr] F14.proof 14 [optionally include the 54-digit factor]
```
is very easy to understand and see what is happening.

Reply:
Actually, I omitted the F14.proof file from the main branch on purpose for a few reasons:

1. It is very quick and easy to download from your site. From README.md just two clicks: one to get to your site and one to download the file. I just did it and it took about 5 seconds!
2. The download time saved is trivial compared to the other steps required to create a runnable cofact binary. In particular: download, build, install and test GMP, then download, build and test mprime / gwnum, then create links and build cofact.
3. I **want** people to go to your site, become familar with it's contents, and download many/all the proofs to be ready to test the new cofactor after the next F12 - F29 factor is found. If they do this over a period of time (rather than all at once when the next factor is found), this helps address the site bandwidth concern you mentioned. Omitting the F14.proof file encourages them to go to your site.
 
I see you have done the merge into the cxc branch, so I will check over my changes then do the merge into main.